packages/sdk/src/oracle/services/skinnyOptimisticOracle.ts

```diff
@@ -0,0 +1,159 @@
+import assert from "assert";
```

Comment:
in the future this is the only file + a factory you will really need to add for more oracles (unless the interface isnt sufficient)